f8a_report/graph_report_generator.py

```diff
@@ -48,7 +46,10 @@ def rectify_latest_version(incorrect_list, eco, stack_flow=False):
 
     try:
         _logger.info("Calling sync_latest_version API for the ecosystem {}".format(eco))
-        _session.post(_SYNC_API_URL, json=deps)
+        resp = requests.post(_SYNC_API_URL, json=deps)
```

Comment:
Is it ok that the call can completely block the workflow? w/o any timeout?

Reply:
that was the idea with which future requests was used earlier. Need to check on this.

Reply:
@tisnik We currently have a gateway timeout of 3600 for the data importer API. So this req wont be dangling forever. And because this reporting is a one time job (runs once every 24 hrs), i feel that it should be ok even if it has to wait for some time. WDYT? If you are ok with that, i will merge this PR
cc: @miteshvp

Reply:
@yzainee you can provide a timeout option in requests.post as a param. 
I feel 20 minutes is a good option. But again check with past data to arrive at timeout